content/go/concepts/strings/terms/hassuffix/hassuffix.md

```diff
@@ -0,0 +1,72 @@
+---
+Title: 'HasSuffix()'
+Description: 'Checks if a given suffix exists on the specified string. Returns true if the string has the given suffix, else it returns false.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Functions'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/computer-science'
+---
+
+The **`HasSuffix()`** function in Go is used to check if a specified suffix exists on a given string. The function returns true if the given string has the specified suffix, otherwise it returns false.
```

Comment:
```suggestion
The **`HasSuffix()`** function in Go is used to check if a specified suffix exists on a given string. The function returns `true` if the given string has the specified suffix, otherwise it returns `false`.
```